### DISCUSSION TITLE
crud operation

### DISCUSSION THREAD

**Discussion by amanborka** [wailsapp/wails#2508] [Q&A]
Is there a way to perform the crud operation in backend Go from frontend  and save data in the sql database and display it again in frontend ?

I apologize that my English is not good

**Comment by OlegGulevskyy** [comment]
Of course, you can "expose" any function from Go to the frontend, where frontend will know which function to call and what it will even return. And in the backend - you can do whatever operation you need, update your data in SQL, make HTTP requests etc.
I'd recommend check out the documentation about Wails and check templates section where you might find some examples of the communication (frontend -> <- backend) usage.
